desktop/assets/texts/de-DE/settings.ftl

```diff
@@ -3,6 +3,7 @@ player-settings = Spieler-Einstellungen
 movie-parameters = Filmparameter
 custom-base-url = Benutzerdefinierte Base-URL
 spoof-swf-url = Gefälschte SWF-URL
+referer-url = Referenz URL
```

Comment:
Hmm: https://de.wikipedia.org/wiki/Referrer

Which one do you think is more appropriate, @evilpie?